### DISCUSSION TITLE
Large language medical models

### DISCUSSION THREAD

**Discussion by chimezie** [OoriData/OgbujiPT#27] [Ideas]
SNOMED-CT is an international medical terminology system and standard of medical concepts, their relationships with each other, and their definitions specified in formal logical semantics as well as some domain expert textual definitions.  These definitions can be used as context for querying an LLM about a subset of medical concepts.

I will explore how this might work with chat_web_selects as a basis for getting up to speed with things

**Comment by uogbuji** [comment]
@chimezie have you seen some of Kingsley's work on using ChatGPT to produce & apply RDF from DBPedia? I wonder how well this could work with private models, as well. Surely similar could apply with SNOWMED-CT.

* https://community.openlinksw.com/t/howto-generating-instant-dbpedia-lookups-and-interactions-using-chat-gpt/3812
* https://community.openlinksw.com/t/howto-linked-data-reasoning-inference-exercise-using-chat-gpt-version-4-engine/3786/3

**Reply by chimezie** [reply]
I spoke with him about a month and a half ago about what he is doing with LLM and SPARQL.  I wasn't very impressed back then, to be honest.  It seemed very abstract and not concrete in terms of value proposition v.s. cool idea.  I'll check these out though.
